test/kalevala/foreman_test.exs

```diff
@@ -35,6 +35,17 @@ defmodule Kalevala.ForemanTest do
 
       assert state.controller == ExampleController
     end
+
+    test "resets the flash state" do
+      conn = setup_conn()
+      state = setup_state()
+
+      conn = Conn.put_flash(conn, :key, "value")
+      conn = Conn.put_controller(conn, ExampleController)
+      {:noreply, state, {:continue, :init_controller}} = Foreman.handle_conn(conn, state)
```

Comment:
I'm not quite sure this is actually testing anything, suggestions on how to test

Reply:
Great question, based on what you have I'm pretty satisfied with it for now. There isn't a ton of testing in place which I'm (very) slowly getting around to.